### PR TITLE
PLFM-6394

### PR DESCRIPTION
**Pull request by brucehoff**

### CODE REVIEW

File: services/repository/src/main/java/org/sagebionetworks/auth/controller/AuthenticationController.java [hunk 1]
```diff
@@ -369,10 +371,12 @@ public void unbindExternalAliasFromAccount(
 	@RequestMapping(value = UrlHelpers.AUTH_PERSONAL_ACCESS_TOKEN, method = RequestMethod.POST)
 	public @ResponseBody AccessTokenGenerationResponse createPersonalAccessToken(
 			@RequestParam(value = AuthorizationConstants.USER_ID_PARAM) Long userId,
+			@RequestHeader(value = AuthorizationConstants.SYNAPSE_AUTHORIZATION_HEADER_NAME, required=false) String authorizationHeader,
```
Comment (marcomarasca) [inline services/repository/src/main/java/org/sagebionetworks/auth/controller/AuthenticationController.java:374]: Might be worth adding this details in the API docs above.

File: services/repository-managers/src/test/java/org/sagebionetworks/repo/manager/authentication/PersonalAccessTokenManagerImplUnitTest.java [hunk 2]
```diff
@@ -131,13 +146,47 @@ void testIssueToken() {
 		verify(mockPersonalAccessTokenDao).deleteLeastRecentlyUsedTokensOverLimit(userInfo.getId().toString(), EXPECTED_TOKEN_LIMIT);
 	}
 
+	@Test
+	void testIssueTokenLimitedAccessTokenScope() {
```
Comment (marcomarasca) [inline services/repository-managers/src/test/java/org/sagebionetworks/repo/manager/authentication/PersonalAccessTokenManagerImplUnitTest.java:150]: You might want to add the test to check on the removeAll for double authorize in the list edge case

**Review by marcomarasca — APPROVED** [review]
